src/components/Login/Login.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { fireEvent, render, screen, waitFor } from "@testing-library/react";
```

Comment:
There is no test to check the error. please write tests for the failing and negative conditions.

Reply:
addressed the comment.